charts/opentelemetry-demo/values.yaml

```diff
@@ -425,6 +425,14 @@ components:
     resources:
       limits:
         memory: 50Mi
+    # additionalVolumes:
+    #   - name: nginx-logs
+    #     hostPath:
+    #     path: /var/log/nginx
+    #     type: ""
+    # extraVolumeMounts:
+    #   - name: nginx-logs
+    #     mountPath: /var/log/nginx
```

Comment:
Please update the comment at the `components:` field that lists all the available fields